src/parser/paladin/retribution/modules/core/azeritetraits/DivineRight.js

```diff
@@ -6,13 +6,10 @@ import SPELLS from 'common/SPELLS';
 import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
 
 
-const divineRightStats = traits => Object.values(traits).reduce((obj, rank) => {
-  const [strength] = calculateAzeriteEffects(SPELLS.DIVINE_RIGHT.id, rank);
-  obj.strength += strength;
-  return obj;
-}, {
-  strength: 0,
-});
+const divineRightStats = traits => Object.values(traits).reduce((strengthSum, trait) => {
+  const [strength] = calculateAzeriteEffects(SPELLS.DIVINE_RIGHT.id, trait);
+  return strengthSum + strength;
+}, 0);
 
 export const STAT_TRACKER = {
   strength: combatant => divineRightStats(combatant.traitsBySpellId[SPELLS.DIVINE_RIGHT.id]).strength,
```

Comment:
this is still trying to use the object property

Reply:
not quite @Juko8. i meant line 15 is still doing `divineRightStats(...).strength` when it should just be `divineRightStats`. Sorry for the confusion. You can revert 6158ffd